snowexsql/api.py

```diff
@@ -165,10 +191,17 @@ def from_unique_entries(cls, columns_to_search, **kwargs):
     @property
     def all_site_names(self):
         """
-        Return all types of the data
+        Return all campaign names
```

Comment:
Does this method signature need to be updated?
Right now, I would expect to get a list of site names. We could just do a
```sql
Select * from sites
```
With the new structure.

I might be over-simplifying things though. We could also log an issue and add a TODO marker to revisit this.

Reply:
I think it's a good idea to make an issue for that (though I think you're right on the method to select them). I think the issue would help make it clear that we're changing naming convention:
site_names -> campaigns
site_ids -> campaign_sites or something